src/components/inputsList/InputsList.tsx

```diff
@@ -27,7 +27,7 @@ export default InputsList;
 
 const renderTableBody = (inputs) => {
```

Comment:
you dont need this method, this file small and obvious just inline function's body into table